src/vg/civcraft/mc/civmenu/TOSListener.java

```diff
@@ -20,67 +15,67 @@
 import net.md_5.bungee.api.chat.ClickEvent;
 
 public class TOSListener implements Listener {
-	
+
 	private CivMenu plugin = CivMenu.getInstance();
-	
+
 	public TOSListener() {
-		
+
 	}
-	
+
 	@EventHandler
 	public void playerJoinEvent(PlayerJoinEvent event) {
 		Player p = event.getPlayer();
-		if (!TOSManager.registeredPlayers
-				.containsKey(p.getUniqueId())) {
+		if (!TOSManager.registeredPlayers.containsKey(p.getUniqueId())) {
 			sendTOS(p);
 		}
 	}
-	
+
 	@EventHandler
-	public void playerMoveEvent(PlayerMoveEvent event){
+	public void playerMoveEvent(PlayerMoveEvent event) {
 		Location from = event.getFrom();
         Location to = event.getTo();
 
         if (from.getBlockX() == to.getBlockX()
-                && from.getBlockY() == to.getBlockY()
```

Comment:
It feels like some sort of a weird lag when you can't jump either. I can add it back if you insists but they cant use jumps to move forward anyway.